persistent/picklecache.py

```diff
@@ -11,16 +11,22 @@
 # FOR A PARTICULAR PURPOSE.
 #
 ##############################################################################
+from __future__ import print_function
```

Comment:
Debugging leftover, sorry. (Is there any better way to debug doctests than with extra print statements?)